### PR TITLE
JW-348 Apply Sagar's patch to linker errors

### PR DESCRIPTION
**Pull request by m-kostrzewa**
Patch taken from here: https://github.com/sagarkchitnis/windows/commit/ff90f9b79097a7df3d788489cd69d0c2e310005a

It was discussed during the meeting.

### CODE REVIEW

**Review by mclapinski — APPROVED** [review]

File: rules.py [hunk 1]
```diff
@@ -1098,11 +1098,11 @@ def SetupBuildEnvironment(conf):
     env['CC'] = 'cl.exe'
     env['CXX'] = 'cl.exe'
     env['TARGET_ARCH']='x86'
-    env['CCPDBFLAGS'] = '/Zi /Fd${TARGET}.pdb'
+    env['CCPDBFLAGS'] = '/Z7'
     opt_level = env['OPT']
     env['OBJSUFFIX'] = '.o'
 	# /GS /analyze- /W3 /Zc:wchar_t /ZI /Gm /Od /Fd"Debug\vc140.pdb" /Zc:inline /fp:precise /D "WIN32" /D "_DEBUG" /D "_CONSOLE" /D "_UNICODE" /D "UNICODE" /errorReport:prompt /WX- /Zc:forScope /RTC1 /Gd /Oy- /MDd /Fa"Debug\" /EHsc /nologo /Fo"Debug\" /Fp"Debug\compilerproj.pch" 
-    env.Append(CCFLAGS = '/Iwindows /GS /analyze- /w /Zc:wchar_t /Gm /Od /Zc:inline /fp:precise /D_WIN32_WINNT=0x0601 /D "_WINDOWS" /D "WIN32" /D "CURL_STATICLIB" /D "_DEBUG" /D "_CONSOLE" /errorReport:prompt /WX- /w /wd4996 /wd4150 /Zc:forScope /RTC1 /Gd /Oy- /MDd /Fa"Debug\" /EHsc ')
```
Comment (Attil) [inline rules.py:1105]: Wouldn't that make the compilation process super long?

**Comment by m-kostrzewa** [comment]
Trying to run agent build from this branch

**Comment by m-kostrzewa** [comment]
http://ci.wincontrail.codilime.com:8080/job/Wincontrail/job/build-vrouter-agent/43/console

@Attil, should be good to merge?

**Review by Attil — APPROVED** [review]

**Comment by m-kostrzewa** [comment]
Janie, przetestuj proszę

**Comment by m-kostrzewa** [comment]
Running build again, after fix was merged to contrail-vrouter

**Comment by m-kostrzewa** [comment]
@Attil I get the following when building:
```cl : Command line warning D9007 : '/Gm' requires '/Zi or /ZI'; option ignored```

http://ci.wincontrail.codilime.com:8080/job/Wincontrail/job/build-vrouter-agent/44/console

**Pull request merged**